pages/content-ui/script/entryPoints.ts

```diff
@@ -0,0 +1,52 @@
+import { readdirSync, existsSync, mkdirSync, writeFileSync, Dirent, readFileSync, renameSync } from 'fs';
+import { resolve } from 'node:path';
+
+const rootDir = resolve(__dirname, '..');
+const srcDir = resolve(rootDir, 'src');
+
+export const getEntryPoints = (options: { include?: string[]; exclude?: string[]; createRecursively?: boolean }) => {
+  const entryPoints: Record<string, string> = {};
+
+  const foldersToProcess =
+    options.include ||
+    readdirSync(srcDir, { withFileTypes: true })
+      .filter((item): item is Dirent => item.isDirectory())
+      .map(item => item.name);
+
+  foldersToProcess.forEach(folderName => {
+    const shouldExclude = options.exclude && options.exclude.includes(folderName);
```

Comment:
Maybe sth like that:

```suggestion
    const shouldExclude = options.exclude?.includes(folderName);
```